src/ui_framework/components/button/_button.scss

```diff
@@ -80,23 +80,23 @@
   }
 }
 
+/**
```

Comment:
I've found it's better to keep comments local to classes (even if they're duplicated), or else it can become really hard to keep track of them as the number of comments and classes grows.